### DISCUSSION TITLE
The road name direction keep change when zoom in/out

### DISCUSSION THREAD

**Discussion by crablife** [maplibre/maplibre-native#831] [Q&A]
I have my own map tile, and in some road, the road name direction will change when zoom in/out map.
Does anyone know what possible reason can cause this issue?
Here is the issue video: https://youtube.com/shorts/zTqfQV4a23c?feature=share
Is this issue cause by map tile or maplibre?

**Comment by ovivoda** [comment]
@crablife can you please add more informations on the maplibre version and the map tiles that you are using for the project?
